test/Rings/solving-test.jl

```diff
@@ -15,8 +15,13 @@
     @test res[2].param[1] == -1 * p1
     @test res[2].param[2] == -1 * p2
 
+    rat_sols = Vector{fmpq}[[1, 0, 0], [1//3, 0, 1//3]]
+    rat_res = Oscar._rational_solutions(I)
+    @test rat_res == rat_sols
```

Comment:
```suggestion
    @test length(rat_res) == 2 && issetequal(rat_res, rat_sols)
```
To avoid problems with the ordering.